src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const dayCost = 40;
+  const weekDiscount = 50;
+  const threeDaysDiscount = 20;
+  const priceWithoutDiscout = days * dayCost;
+  let price = priceWithoutDiscout;
+
+  if (days >= 3 && days < 7) {
+    price = price - threeDaysDiscount;
+  }
+
+  if (days >= 7) {
```

Comment:
You can also use **if** and **else if**.
```

  if (days >= 7) {
    --- code
  } else if (days >= 3) {
    --- code
  }

Reply:
Read the checklist, point 4.